composer.json

```diff
@@ -9,15 +9,14 @@
     "require": {
         "php": ">=5.3.3",
         "symfony/symfony": "2.3.*",
-        "doctrine/orm": "~2.2,>=2.2.3,<2.5",
-        "doctrine/dbal": "<2.5",
+        "doctrine/orm": "^2.4.8",
         "doctrine/doctrine-bundle": "~1.2",
         "twig/extensions": "1.0.*",
```

Comment:
this one should be changed to ``~1.0`` to allow to use uptodate versions (the latest stable is 1.2.0)